src/test/java/uk/gov/companieshouse/api/testdata/service/impl/OfficerAppointmentServiceImplTest.java

```diff
@@ -73,11 +83,50 @@ void create() throws DataException {
         assertEquals(3, officerAppointment.getDateOfBirthMonth().intValue());
         assertEquals("Test DIRECTOR", officerAppointment.getName());
 
-        assertOfficerItem(officerAppointment.getOfficerAppointmentItems().get(0));
+        assertOfficerItem(officerAppointment.getOfficerAppointmentItems().get(0), "Wales");
+
+    }
+
+    @Test
+    void createScotland() throws DataException {
+        Address mockAddress = new Address();
+        CompanySpec spec = new CompanySpec();
+        spec.setCompanyNumber(COMPANY_NUMBER);
+        spec.setJurisdiction(Jurisdiction.ENGLAND_WALES);
```

Comment:
Was this meant to be Scotland? 
This works because we are not actually doing anything with the jurisdiction other than sending it to the address service and we are mocking that behaviour so we don't really care about the jurisdiction. Maybe we don't need different test cases then?